code/game/machinery/launch_pad.dm

```diff
@@ -192,35 +192,24 @@
 		return ..()
 
 //Briefcase item that contains the launchpad.
-/obj/item/briefcase_launchpad
-	name = "briefcase"
-	desc = "It's made of AUTHENTIC faux-leather and has a price-tag still attached. Its owner must be a real professional."
-	icon = 'icons/obj/storage.dmi'
-	icon_state = "briefcase"
-	lefthand_file = 'icons/mob/inhands/equipment/briefcase_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/equipment/briefcase_righthand.dmi'
-	flags_1 = CONDUCT_1
-	force = 8
-	hitsound = "swing_hit"
-	throw_speed = 2
-	throw_range = 4
-	w_class = WEIGHT_CLASS_BULKY
-	attack_verb = list("bashed", "battered", "bludgeoned", "thrashed", "whacked")
-	resistance_flags = FLAMMABLE
-	max_integrity = 150
+/obj/item/storage/briefcase/launchpad
 	var/obj/machinery/launchpad/briefcase/pad
 
-/obj/item/briefcase_launchpad/Initialize()
+/obj/item/storage/briefcase/launchpad/Initialize()
 	. = ..()
 	pad = new(src)
 
-/obj/item/briefcase_launchpad/Destroy()
+/obj/item/storage/briefcase/launchpad/Destroy()
 	if(!QDELETED(pad))
 		qdel(pad)
 	pad = null
```

Comment:
You can probably replace all this with QDEL_NULL(pad)

Reply:
I don't know why, but replacing it causes a qdel loop

Reply:
keep the !QDELETED(pad) check, now that i remember it was necessary due to the pad deleting the briefcase if destroyed and viceversa